From 33456bbc3d0332bd2e071c547303115f83dbfcf9 Mon Sep 17 00:00:00 2001
From: liyong4x <yongx.a.li@intel.com>
Date: Tue, 12 Jun 2018 04:16:37 -0400
Subject: [PATCH 091/129] media: intel-ipu4: pdata: Correct ar0231at input clk
 and pclk

Message for Open Source:
The previous input clk and pclk value can work normally,
but not recommended. So change as the right values from
sensor vendor.

Message for Internal:
The previous input clk and pclk value can work normally,
but not recommended. So change as the right values from
sensor vendor.

[Issue/Feature] The previous input clk and pclk value are
not recommended by vendor.

[Root Cause/Changes]
1. change the input_clk as 27000000
2. change the op_sys_clk as 87750000

Change-Id: I4b265670ad098c2447aece7335accc6a2f61ae69
Tracked-On: #JIIAP-475
Signed-off-by: liyong4x <yongx.a.li@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/platform/intel/ipu4-bxt-p-pdata.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/platform/intel/ipu4-bxt-p-pdata.c b/drivers/media/platform/intel/ipu4-bxt-p-pdata.c
index 334b105..d6fab2a 100755
--- a/drivers/media/platform/intel/ipu4-bxt-p-pdata.c
+++ b/drivers/media/platform/intel/ipu4-bxt-p-pdata.c
@@ -962,8 +962,8 @@
 
 static struct crlmodule_platform_data ar0231at_pdata = {
 	.lanes = AR0231AT_LANES,
-	.ext_clk = 24000000,
-	.op_sys_clock = (uint64_t[]){ 264000000 },
+	.ext_clk = 27000000,
+	.op_sys_clock = (uint64_t[]){ 87750000 },
 	.module_name = "AR0231AT",
 };
 #endif
-- 
1.9.1

